server/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.8-slim-buster
+
+WORKDIR /server
+
+# Copy the code of the server directory to /server of the machine
+COPY . .
+
+# install flask
+RUN pip install flask flask-cors
+
+#Expose container port 5000
+EXPOSE 5000
+
+CMD ["python3", "-m" , "flask", "run", "--host=0.0.0.0"]
```

Comment:
I added a script in the `pipfile` to run the app, `pipenv run start` , it's also using nodemon so I am not sure if we should add a new script that is used to run the app in the container, or we should just install nodemon